src/main/java/it/gov/pagopa/payment/repository/TransactionInProgressRepositoryExtImpl.java

```diff
@@ -217,4 +220,35 @@ public List<TransactionInProgress> findByFilter(Criteria criteria, Pageable page
     public long getCount(Criteria criteria) {
         return mongoTemplate.count(Query.query(criteria), TransactionInProgress.class);
     }
+
+    @Override
+    public TransactionInProgress findCancelExpiredTransaction() {
```

Comment:
let's explicitly suggest that this method is Throttled, adding it as suffix to method name